drv/lsm6ds/lsm6ds.c

```diff
@@ -0,0 +1,155 @@
+/**
+ * @file lsm6ds.c
+ * @author Mališa Vučinić <malisa.vucinic@inria.fr>
+ * @brief Driver for controlling the LSM6DS IMU.
+ *
+ * @copyright Inria, 2023
+ *
+ */
+
+#include <assert.h>
+#include <math.h>
+#include <stdio.h>
+
+#include "gpio.h"
+#include "i2c.h"
+#include "lsm6ds.h"
+
+//=========================== defines ==========================================
+
+static const gpio_t scl     = { .port = 1, .pin = 9 };
+static const gpio_t sda     = { .port = 0, .pin = 11 };
```

Comment:
```suggestion
static const gpio_t scl     = { .port = DB_I2C_SCL_PORT, .pin = DB_I2C_SCL_PIN };
static const gpio_t sda     = { .port = DB_I2C_SDA_PORT, .pin = DB_I2C_SDA_PIN };
```
(and run make format to align with the line below)

Reply:
fixed